concepts/comparisons/links.json

```diff
@@ -1,18 +1,54 @@
 [
   {
-    "url": "http://example.com/",
-    "description": "TODO:  add new link (above) and write a short description here of the resource."
+    "url": "https://docs.python.org/3/reference/expressions.html#comparisons",
+    "description": "Comparisons in Python (Python language reference)"
   },
   {
-    "url": "http://example.com/",
-    "description": "TODO:  add new link (above) and write a short description here of the resource."
+    "url": "https://www.tutorialspoint.com/python/python_basic_operators.htm",
+    "description": "Python basic operators on tutorialspoint"
   },
   {
-    "url": "http://example.com/",
-    "description": "TODO:  add new link (above) and write a short description here of the resource."
+    "url": "https://data-flair.training/blogs/python-comparison-operators/",
+    "description": "Python comparison operators on data-flair"
   },
   {
-    "url": "http://example.com/",
-    "description": "TODO:  add new link (above) and write a short description here of the resource."
+    "url": "https://www.python.org/dev/peps/pep-0207/",
+    "description": "PEP 207 to allow Operator Overloading for Comparison"
+  },
+  {
+    "url": "https://docs.python.org/3/reference/expressions.html#is-not",
+    "description": "Identity comparisons in Python (Python language reference)"
+  },
+  {
+    "url": "https://docs.python.org/3/library/operator.html",
+    "description": "Python operators official doc"
```

Comment:
The descriptions for the python.org links are inconsistent.

Reply:
Thats a nice pick! I have updated this one. Do you think all of them look good or we need to change anything?

Reply:
@gurupratap-matharu - these are fine as is.  There is no rule they have to be consistent.

Reply:
Okay. Resolving this conversation